lib/container_server.go

```diff
@@ -467,7 +467,9 @@ func (c *ContainerServer) LoadSandbox(id string) error {
 		}
 	}
 
-	c.ContainerStateFromDisk(scontainer)
+	if err := c.ContainerStateFromDisk(scontainer); err != nil {
```

Comment:
These changes have direct influence on the current unit tests, which will now fail because the `state.json` is not available. :-/

Reply:
Is there a way to add state.json files for every sandbox and container created in the test framework?

Reply:
Yes, this should work. We could create a new test function in `lib/suite_test` like this:

```go
func createDummyState() {
	ioutil.WriteFile("state.json", []byte("{}"), 0644)
}
```

Then we could use this function for all seven failing tests to succeed. More should not be needed, since every test already cleans up via `removeState()`.